### ISSUE TITLE
Problem with the quickstart script

### ISSUE THREAD

**Issue by marcel1309** [InstaPy/InstaPy#5735]
selenium.common.exceptions.SessionNotCreatedException: Message: Unable to find a matching set of capabilities thats my error code idk how to fix it

**Comment by mastino92** [comment]
i have something similar with this error first time i'm uising it and get this 
```
Traceback (most recent call last):
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\Quickstart.py", line 52, in <module>
    session = InstaPy(username=insta_username,
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\instapy\instapy.py", line 312, in __init__
    self.browser, err_msg = set_selenium_local_session(
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\instapy\browser.py", line 121, in set_selenium_local_session
    browser = webdriver.Firefox(
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\selenium\webdriver\firefox\webdriver.py", line 170, in __init__
    RemoteWebDriver.__init__(
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 252, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\Cpu\AppData\Local\Programs\Python\Python38-32\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Unable to find a matching set of capabilities
```

**Comment by Abdulsamipy** [comment]
add geckodriver.exe to the path and add the geckodriver in the same folder as quickstart.py

**Comment by mastino92** [comment]
> add geckodriver.exe to the path and add the geckodriver in the same folder as quickstart.py
wich path? i've put it everything still get errors :(

**Comment by rnmhdn** [comment]
gecko driver is in my path but I still get the error. I tried with both releases 0.26 and 0.27. I tried with both manually setting the browser executable path and automatic path. nothing works. the only thing I could get to work was using firefox gui.

```
$ ./geckodriver --version
geckodriver 0.27.0 (7b8c4f32cdde 2020-07-28 18:16 +0000)
$ which geckodriver 
/usr/bin/geckodriver
$ geckodriver --version
geckodriver 0.26.0
```
I tried setting `browser_executable_path` to global path, relative to local path to the same place as the python script, absolute path to local isntall etc. nothing works. I always get the same error.
